README.md

```diff
@@ -1,69 +1,61 @@
-# go-gh beta
+# Go library for the GitHub CLI
 
-**This project is in beta!** Feedback is welcome.
+`go-gh` is a collection of Go modules to make authoring [GitHub CLI extensions][extensions] easier.
 
-A Go module for CLI Go applications and [gh extensions][extensions] that want a convenient way to interact with [gh][], and the GitHub API using [gh][] environment configuration.
+Modules from this library will obey GitHub CLI conventions by default:
 
-`go-gh` supports multiple ways of getting access to `gh` functionality:
+- [`CurrentRepository()`](https://pkg.go.dev/github.com/cli/go-gh#CurrentRepository) respects `GH_REPO` and reads from git remote configuration as fallback;
```

Comment:
nit: I don't often see semicolons at the end of a list item, it's usually either a `.` or empty